cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -14,14 +14,41 @@
 
 from typing import (Any, Dict, Iterable, List, NamedTuple, Optional, Sequence,
                     Set, Tuple, Union)
+import dataclasses
 import numpy as np
+import scipy
 from matplotlib import pyplot as plt
 from cirq import circuits, devices, ops, protocols, sim, work
 
 CrossEntropyPair = NamedTuple('CrossEntropyPair', [('num_cycle', int),
                                                    ('xeb_fidelity', float)])
 
 
+@dataclasses.dataclass
+class DepolarizingModel:
+    """A model of depolarizing noise.
+
+    A model of a depolarizing process that maps a density matrix ρ as
+
+        ρ → (S * p^d) ρ + (1 - S * p^d) I / D
+
+    where I / D is the maximally mixed state, S and p are real numbers, and
+    d is the number of iterations of the process. For instance, in a cross
+    entropy benchmarking experiment, d would be the number of cycles, where a
+    cycle consists of a layer of single-qubit gates followed by a layer of
+    two-qubit gates.
+
+    Attributes:
+        coefficient: The value S in the above formula.
+        decay_constant: The value p in the above formula.
```

Comment:
The two parameters (coefficient and decay_constant) can be given better description, e.g.

```
coefficient: depolarization constant for state preparation and measurement, i.e. S in the above formula.
decay_constant: depolarization constant for circuit execution, i.e. p in the above formula.
```

You could also name the two parameters better :-)

Reply:
I've updated the descriptions and renamed the parameters to `spam_depolarization` and `cycle_depolarization`.